src/open_ai.js

```diff
@@ -0,0 +1,13 @@
+const { Configuration, OpenAIApi } = require('openai');
+
+const OPEN_AI_IMPROVE_CHANGELOG_REQUEST = `You are an expert programmer, and you are trying to rewrite release changes into user-friendly text.
+For each line bellow write one meaningful past infinitive sentence, starting each bullet point with a \`* \`, and the sentence ends with \`.\`.`;
+
+const openaiConfiguration = new Configuration({
+    apiKey: process.env.OPEN_AI_TOKEN,
```

Comment:
I'd use action inputs, instead of env vars. It is more of a standard.

Reply:
I use env var as it is an experiment. You can simply remove env var if you don't like it. With action input you need to do a breaking change.

Reply:
What breaking change? I'd simply create input that is not required. You can treat it the same as any other variable.
That's not breaking.

Reply:
I mean, if you pass env var, which code didn't use you will be fine. But if you pass input which action didn't have in action.yaml, you got an error.

But anyway, the input make more sense, so I updated it, just leave env var to be able test it locally.

Reply:
Cool. Just FYI inputs are in fact env vars (see [here](https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L128)), so you can use it locally to supply an input.

For Example: `open-ai-token` -> `INPUT_OPEN_AI_TOKEN`.